src/config/types.ts

```diff
@@ -128,23 +128,16 @@ export interface ExpectOptsConfig {
     interval: number;
 }
 
-export interface MochaOpts {
-    /** milliseconds to wait before considering a test slow. */
-    slow?: number;
-
-    /** timeout in milliseconds or time string like '1s'. */
-    timeout?: number;
-
-    /** string or regexp to filter tests with. */
-    grep?: string | RegExp;
+export interface MochaOpts extends Omit<Mocha.MochaOptions, "ui"> {
+    ui?: string | ((suite: Mocha.Suite) => void);
 }
 
 export interface SystemConfig {
     debug: boolean;
     mochaOpts: MochaOpts;
     expectOpts: ExpectOptsConfig;
     ctx: { [name: string]: unknown };
-    patternsOnReject: Array<string>;
+    patternsOnReject: Array<string | RegExp>;
```

Comment:
It can be specified as RegExp. Like this:
```
patternsOnReject: [/foo/i, /bar/, /baz/g]
```

Under the hood we wrap all items from `patternsOnReject` to RegExp. Like this - `new RegExp(pattern)`. So it works correctly if user specify regexp value.